frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -26,6 +35,25 @@ interface OperatorSelectProps extends SelectProps<any> {
     defaultOpen?: boolean
 }
 
+function getValidationError(operator: PropertyOperator, value: any, property?: string): string | null {
+    if (isOperatorRegex(operator)) {
+        try {
+            new RegExp(value)
+        } catch (e: any) {
+            return e.message
+        }
+    }
+    if (isOperatorRange(operator) && isNaN(value)) {
+        let message = `Range operators only work with numeric values`
+        if (dayjs(value).isValid()) {
+            const propertyReference = property ? `property ${property}` : 'this property'
+            message += `. If you'd like to compare dates and times, make sure ${propertyReference} is typed as DateTime in Data Management. You will then be able to use operators "before" and "after"`
```

Comment:
For extra 🦔 points we could link to the right part of data management here... But I love that the validation error gives them a way to fix the error

Reply:
Kept this string-only to :ship: quicker!